demo/src/main/java/com/google/googlemediaframeworkdemo/demo/adplayer/ImaPlayer.java

```diff
@@ -297,7 +299,7 @@ public void removeCallback(VideoAdPlayerCallback videoAdPlayerCallback) {
      * then the start method will be ignored.
      */
     @Override
-    public VideoProgressUpdate getProgress() {
+    public VideoProgressUpdate getAdProgress() {
```

Comment:
Yes, it is. VideoAdPlayer now extends AdProgressProvider which provides the newly-named method and the old method is gone.